.github/workflows/e2e-tests-workflow.yml

```diff
@@ -8,10 +8,7 @@ on:
       - main
 env:
   OPENSEARCH_DASHBOARDS_VERSION: '1.x'
-  AD_OPENSEARCH_DASHBOARDS_PLUGIN_NAME: anomaly-detection-dashboards
-  OPENSEARCH_DOCKER_IMAGE: opensearchstaging/opensearch
-  DASHBOARDS_DOCKER_IMAGE: opensearchstaging/opensearch-dashboards
-  DOCKER_TAG: 1.1.0
+  OPENSEARCH_VERSION: '1.2.0-SNAPSHOT'
 jobs:
   test-with-security:
     name: Run e2e tests with security
```

Comment:
the regression here is now we're only testing with no security (this can be renamed). Part of the reason for using the docker images was that security was included. Is there a way to pull security dependencies from maven?

Reply:
Looking into it